internal/nexus_operations.go

```diff
@@ -174,6 +174,29 @@ func apiOperationErrorToNexusOperationError(opErr *nexuspb.UnsuccessfulOperation
 	}
 }
 
+func apiHandlerErrorToNexusHandlerError(apiErr *nexuspb.HandlerError, failureConverter converter.FailureConverter) (*nexus.HandlerError, error) {
```

Comment:
Add a comment that this is a copy from the server code.
I wonder if it's possible to share this code.

Reply:
The whole block of functions is surrounded by a comment that says it's copied from the server code.
I'm hesitant to share to avoid exporting these functions from the SDK.